libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -28,19 +28,22 @@ public static NewSlackMessage ActivityToSlack(Activity activity)
 
```

Comment:
Question: Shouldn't we throw an Exception if `Activity` is null?

Reply:
Yes, you are right. Thanks!

Reply:
Sorry, not an exception, just return null.